src/solidation/tests/test_main.py

```diff
@@ -0,0 +1,5 @@
+from ..__main__ import sanitize_md
```

Comment:
Move this file to `tests/test_main.py` relative to the project root (i.e., outside of `src/`).

Reply:
I really see little benefits and only inconveniences from such separation, but ok, will do,  since you are the lead on this one.

Reply:
done in https://github.com/con/solidation/pull/43/commits/4978b8a9099b4f4f1c3e1072efb6ebb330b43722 which also required replicating listing of tests now for all other `tox` components which take care about source code